### DISCUSSION TITLE
Converting Form components from laravelcollective/html to spatie/laravel-html

### DISCUSSION THREAD

**Discussion by DamianDeMie** [spatie/laravel-html#205] [Q&A]
Hello,

My company is in the middle of migrating from laravelcollective/html to spatie/laravel-html, however we use quite a few custom components from the old package and are having issues converting it to the new laravel-html syntax.

An example of what we have is

`        Form::component('imText', 'general.formfields.text', ['label', 'name', 'value', 'attributes']);
`

We want to be able to use a Blade component like this, give it a name and attributes, how would we go about doing that in laravel-html?

**Comment by levaconline** [comment]
I'm not sure is it possible in laravel-html, but if you cannot find solution you can always try with  native larave's component: 
https://readerstacks.com/how-to-make-a-component-in-laravel-10/
https://laravel.com/docs/10.x/blade#rendering-components
